### ISSUE TITLE
One to one relationship has incorrect table.NameHumanCase on Configuration/Mapping generation

### ISSUE THREAD

**Issue by FaizulHussain** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#716]
Hi,

I think there might be an issue with One2One relationship reverse mapping.

If I have these tables:
```sql
CREATE TABLE [dbo].[A]
(
    [Id] UNIQUEIDENTIFIER PRIMARY KEY,
	[Name] NVARCHAR(256)
)
GO
CREATE TABLE [dbo].[B]
(
    [Id] UNIQUEIDENTIFIER PRIMARY KEY FOREIGN KEY REFERENCES [dbo].[A] ([Id]),
	[Description] NVARCHAR(MAX)
)
GO
```
With rename table handler adding suffix like "Entity", then it generates the mapping as:
```cs
// Foreign keys
builder
   .HasOne(a => a.B)
   .WithOne(b => b.A)
   .HasForeignKey<**B**>(c => c.Id)
   .OnDelete(DeleteBehavior.ClientSetNull)
   .HasConstraintName("FK_dbo_B_Id_A");
```
The class names are correctly generated with the renamed suffix (A**Entity**, B**Entity** etc...) 
```cs
case Relationship.OneToOne:
return string.Format(".WithOne({0}){1}.HasForeignKey<{2}>({3})",
   withParam, hasPrincipleKey, **fkTableNameHumanCase**, manyToManyMapping);
```
Thanks

**Comment by FaizulHussain** [comment]
I changed the line 4851 
```cs
var rel = GetRelationship(relationship, fkCols2, pkCols2, pkPropName, fkPropName, manyToManyMapping, mapKey, foreignKey.CascadeOnDelete, foreignKey.IncludeReverseNavigation, foreignKey.IsNotEnforced, foreignKey.ConstraintName, pkTableHumanCase, fkTable.NameHumanCase**WithSuffix()**, primaryKeyColumns, fkHasUniqueConstraint);
```
that did the trick

Thanks

**Comment by sjh37** [comment]
Thanks @FaizulHussain
This will be in the next release :-)

**Comment by sjh37** [comment]
Now released in v3.5.0
